internal/common/enums.rs

```diff
@@ -341,6 +341,8 @@ macro_rules! for_each_enums {
                 Spinbox,
                 /// The element is a [`Tab`](../widgets/tabwidget.md) or behaves like one.
                 Tab,
+                /// The element is a [`TabList`](../widgets/tabwidget.md) or behaves like one.
+                TabList,
```

Comment:
```suggestion
                /// The element is a [`TabWidget`](../widgets/tabwidget.md) or behaves like one.
                TabList,
```

But even then , this is more like the tabbar.

```suggestion
                /// The element is similar to the tab bar in a [`TabWidget`](../widgets/tabwidget.md).
                TabList,
```